fvm/transactionVerifier.go

```diff
@@ -28,24 +28,44 @@ type signatureEntry struct {
 	flow.TransactionSignature
 
 	signatureType
+}
+
+// signatureContinatuion is an internal/helper struct, accessible only by
+// TransactionVerifier, used to keep track of the signature verification
+// continuation state.
+type signatureContinuation struct {
+	// signatureEntry is the initial input.
+	signatureEntry
 
+	// accountKey is set by getAccountKeys().
 	accountKey flow.AccountPublicKey
 
+	// verified and verifyErr are set by verifyAccountSignatures().  Note that
+	// verifyAccountSignatures() is always called after getAccountKeys()
+	// (i.e., accountKey is always initialized by the time
+	// verifyAccountSignatures is called).
+	verified  bool
```

Comment:
`verified=true` reads as "signature was successfully verified" to me. Which is not what this is for. This is more like: "this signature already went through the verification step".  Maybe a name like `verificationDone` would make that clearer.